index.html

```diff
@@ -1,657 +1,1213 @@
-                            <span class="btn-text">More About Us</span>
-                        </a>
+                  </div>
+
+                  <div class="text aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div id="text-924f8b967f" class="cmp-text">
+                      <p>When leaders are transformed, we affect change through the channels and connections they have – family, friends, co-workers and colleagues. Deep, positive, long-lasting change that can help make our world better.</p>
                     </div>
+                  </div>
+
+                  <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <a id="button-254edd7bf9" class="cmp-button" href="/content/leaderimpact/lang/en2/stories.html">
+                      <span class="cmp-button__text">More About Us</span>
+                    </a>
+                  </div>
                 </div>
-                <div class="icon-box-wrap">
-                    <div class="icon-box-item">
-                        <div data-asset-id="" id="" data-cmp-data-layer="" class="cmp-image" itemscope="" itemtype="">
-                            <img srcset="./assets/article-item.svg" src="./assets/article-item.svg" loading="lazy" class="logo-img" itemprop="contentUrl"  alt="logo" title="logo">
+              </div>
+              <div class="container responsivegrid icon-box-wrap aem-GridColumn aem-GridColumn--default--6 layout-grid">
+                <div id="container-85b30b8ad8" class="cmp-container">
+                  <div class="teaser icon-box-item-col">
+                    <div id="teaser-66129ff9d4" class="cmp-teaser" data-cmp-data-layer="{&quot;teaser-66129ff9d4&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/teaser&quot;,&quot;dc:title&quot;:&quot;Teaser Title&quot;,&quot;dc:description&quot;:&quot;<p>Teaser Description</p>&quot;}}">
+                      <div class="cmp-teaser__image image">
+                        <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_1381991251/container/container/container_copy/image.coreimg{.width}.svg/1669066727362/article-item.svg" data-asset="/content/dam/LeaderImpact/site-./assets/article-item.svg" data-title="logo" id="image-01455c452c" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                          <img src="./assets/article-item.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="logo">
                         </div>
-                        <div class="box-title-wrap" data-cmp-data-layer="" id="">
-                            <h4>Articles</h4>
-                            <p>Lorem ipsum dolor sit amet, consetetur sadip.</p>
+                      </div>
+                      <div class="cmp-teaser__content box-title-wrap ">

```

Comment:
@Prystal254 We can only add custom classes to the root element of components, so `.box-title-wrap` can't go here.